### PR TITLE
Add teaser to ets infinite hot/cold water

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
A new pattern is emerging on having flexibility to connect the models. 

#### What does this PR accomplish?
Add TEASER to the flexibility pattern.

#### How should this be manually tested?
* code review
* `py.test`
* run in dymola, verify output of `teaser_single_new`

#### What are the relevant tickets?
#226

### CODE REVIEW

File: geojson_modelica_translator/model_connectors/couplings/templates/Teaser_EtsColdWaterStub/ComponentDefinitions.mopt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+  Modelica.Blocks.Sources.RealExpression TChiWatSup_{{ coupling.id }}(
+    y=min(
+      {{ coupling.load.id }}.terUni.T_aChiWat_nominal))
+    "Chilled water supply temperature"
+    {% raw %}annotation (Placement(transformation(extent={{-110,-50},{-90,-30}}))){% endraw %};
+
+  // This component model is only needed once even if there is heating and cooling. For now put this on the
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/couplings/templates/Teaser_EtsColdWaterStub/ComponentDefinitions.mopt:7]: @macintoshpie  -- see this and the other comment in the connect statements.mopt. Not sure on how to accurately handle this.

File: geojson_modelica_translator/model_connectors/model_base.py [hunk 2]
```diff
@@ -195,7 +195,12 @@ def render_instance(self, template_params):
         template_env = Environment(
             loader=FileSystemLoader(searchpath=self.template_dir),
             undefined=StrictUndefined)
-        template = template_env.get_template(f'{self.model_name}_Instance.mopt')
+        template_name = f'{self.model_name}_Instance.mopt'
+        try:
+            template = template_env.get_template(template_name)
+        except exceptions.TemplateNotFound:
+            raise Exception(f"Could not find mopt template '{template_name}' in {self.template_dir}")
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/model_base.py:202]: Let the user know where the template file should be located.

File: geojson_modelica_translator/model_connectors/templates/SpawnBuilding.mot [hunk 3]
```diff
@@ -15,7 +15,7 @@ model building
   parameter Integer facSca[nZon]=fill(
     5,
     nZon)
-    "Scaling factor to be applied to on each extensive quantity";
+    "Scaling factor to be applied to each extensive quantity";
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/templates/SpawnBuilding.mot:18]: typo

File: geojson_modelica_translator/model_connectors/templates/Teaser_Instance.mopt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+  // How do we pass this in as a variable
+  parameter String weaName="modelica://teaser_single_new/Loads/Resources/Data/B5a6b99ec37f4de7f94020090/USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.mos"
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/templates/Teaser_Instance.mopt:2]: @macintoshpie -- not sure how to pass in the value from the System Parameter file. Can we discuss tomorrow?

**Review by macintoshpie — APPROVED** [review]
This looks great 👍 

We just need to chat about the comments you made, but I think this can go ahead and get merged for now if we want to keep the ball rolling

py.test passed, the models _look_ right to me and I was able to open the package in Dymola, but I can't run it with my trail version to verify it works as expected.

File: geojson_modelica_translator/model_connectors/templates/TeaserCouplingETS.mot [hunk 5]
```diff
@@ -107,7 +109,7 @@ model TeaserCouplingETS
     pAtm(
       displayUnit="Pa")=101339,
     filNam=ModelicaServices.ExternalReferences.loadResource(
-      "modelica://Buildings/Resources/weatherdata/USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.mos"),
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/templates/TeaserCouplingETS.mot:110]: I can't wait until all these hard coded versions are gone... we missed this quite a while ago.

File: geojson_modelica_translator/model_connectors/couplings/templates/Teaser_EtsColdWaterStub/ComponentDefinitions.mopt (hunk 1 above)
Comment (macintoshpie) [reply geojson_modelica_translator/model_connectors/couplings/templates/Teaser_EtsColdWaterStub/ComponentDefinitions.mopt]: After further discussion, we decided it was best to move the weather data into the load model itself.

**Pull request merged**